### ISSUE TITLE
make mark as junk work properly

### ISSUE THREAD

**Issue by mrexodia** [x64dbg/x64dbg#946]
the data type for junk is present but currently unused

**Comment by torusrxxx** [comment]
This data type is not needed. We can always mark junk bytes as bytes.

**Comment by mrexodia** [comment]
Yeah but we can mark junk code bytes in a different color.

**Comment by blaquee** [comment]
Was this addressed in that patch #1162 ?

**Comment by mrexodia** [comment]
No, that is for those specific instructions, it doesn't use the types
